docs/_docs/04_reference/cli-options.md

```diff
@@ -1306,3 +1306,21 @@ Specify the plan output directory for the `*-all` commands. Useful to save plans
 - [run-all](#run-all)
 
 Specify the output directory for the `*-all` commands to store plans in JSON format. Useful to read plans programmatically.
+
+### terragrunt-auth-provider-cmd
+
+**CLI Arg**: `--terragrunt-auth-provider-cmd`<br/>
+**Environment Variable**: `TERRAGRUNT_AUTH_PROVIDER_CMD`<br/>
+**Requires an argument**: `--terragrunt-auth-provider-cmd /path/to/executable/file`<br/>
+
+The path to the executable file to obtain authentication credentials. If specified, Terragrunt runs this command for every working directory. The output must be the following JSON string
```

Comment:
```suggestion
The path to the executable file to obtain authentication credentials. If specified, Terragrunt runs this command for every working directory.

The output must be valid JSON of the following schema:
```

Then show that it's a map of string to string, and include some keys that are not related to AWS.

Reply:
Updated, thanks!